lib/config.js

```diff
@@ -66,13 +66,14 @@ class Config {
         this.parser = options.parser;
         this.parserOptions = options.parserOptions || {};
 
+        this.configCache = new ConfigCache();
```

Comment:
That's exactly what I've tried earlier today. If you attach a debugger and look at `this.baseConfig` after it's been resolved, you'll see that `extends` haven't been resolved and now there are 3 of them there, instead of 1 that's supposed to be. I don't think that's right.